### ISSUE TITLE
Feature Suggestion: BeatSaber Version Lock

### ISSUE THREAD

**Issue by oliverlorenz** [Assistant/ModAssistant#427]
Hello together,

First of all, I would like to express my absolute appreciation for this tool. I like BeatSaber, but without ModAssistant BeatSaber would only be half as good. Great praise! Of course this thanks also goes to the many mod authors, but your tool makes it smooth and easy. Thanks a lot for your hard work!

It's even worse when BeatSaber releases a new update and suddenly all the mods don't work anymore. I am aware that this is a deliberate protection mechanism to prevent users from breaking their game. And thats great and important!

Nevertheless, I start BeatSaber more and more rarely, because it feels like roulette, whether I can play relaxed today, or I'm busy 1h with fixes downgrading BeatSaber. 

tl;dr;

I would love it if ModAssistant had a feature that could lock and/or reset the Steam version of BeatSaber. Corresponding guides and tools seem to exist as seprate programs. Could you integrate these perhaps to reduce the effect of "suddenly everything is broken"?

I would be pleased about feedback, whether you could imagine this feature in the scope of ModAssistant.

**Comment by Assistant** [comment]
Unfortunately that is outside the scope of Mod Assistant. Downgrading game versions is no longer supported by Beat Saber, and while Mod Assistant will allow you to install mods for any version supported by the backend, sometimes mods that used to work will no longer work on versions they previously worked on, and as such isn't supported by BSMG, the community Mod Assistant is meant to serve.
